index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Landing page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header id="header">
+        <nav id="nav-bar">
+            <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="logo"
+                class="logo">
+            <ul>
+                <a class="nav-link" href="#features">Features</a>
+                <a class="nav-link" href="#how-it-work">How It Works</a>
+                <a class="nav-link" href="#pricing">Pricing</a>
+            </ul>
```

Comment:
Please, your ul tag supposes to have a li tag inside of it, the proper usage of unorder-list usually goes with a child li, before putting the a tag.